proto/configpb.proto

```diff
@@ -0,0 +1,99 @@
+syntax = "proto3";
+package configpb;
+
+import "gogoproto/gogo.proto";
+import "rustproto.proto";
+
+option (gogoproto.sizer_all) = true;
+option (gogoproto.marshaler_all) = true;
+option (gogoproto.unmarshaler_all) = true;
+option (rustproto.lite_runtime_all) = true;
+
+service Config {
+    rpc Create(CreateRequest) returns (CreateResponse) {}
+    rpc Get(GetRequest) returns (GetResponse) {}
+    rpc Update(UpdateRequest) returns (UpdateResponse) {}
+}
+
+message Status {
+    enum StatusCode {
+        UNKNOWN = 0;
+        OK = 1;
+        FAILED = 2;
+        STALE_VERSION = 3;
+    }
+    StatusCode code = 1;
+    string message = 2;
+    Version version = 3;
+}
+
+message Version {
+    uint64 local = 1;
+    uint64 global = 2;
+}
+
+enum Component {
+    UNKNOWN = 0;
+    PD = 1;
+    TiKV = 2;
+    TiDB = 3;
+}
+
+message Local {
+    bytes component_id = 1;
```

Comment:
Seems strings is better.

Reply:
I am curious why it is better.

Reply:
I guess the component identifier should be some readable text?